crates/uv/Cargo.toml

```diff
@@ -103,10 +99,15 @@ tempfile = { workspace = true }
 zip = { workspace = true }
 
 [package.metadata.cargo-shear]
-ignored = ["flate2"]
+ignored = ["flate2", "uv-production-memory-allocator", "uv-production-flate2-backend"]
 
 [features]
-default = ["flate2/zlib-ng", "python", "pypi", "git"]
+default = ["python", "pypi", "git"]
+# Use better memory allocators, etc. — also turns-on self-update.
+production = ["self-update", "production-memory-allocator", "production-flate2-backend", "uv-distribution/production"]
```

Comment:
I think we should keep `self-update` separate, since re-distributors will likely want to run with `--features production`, but _won't_ want to enable `self-update`. `self-update` is only applicable when you install via our dedicated installers, not via `brew`, etc.

Reply:
Are we going to need to include a note to redistributors in the changelog for the `--features production` flag? Does that suggest we should reserve this change for a breaking release?